app/assets/stylesheets/cats.css.scss

```diff
@@ -0,0 +1,58 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+.container#main-container {
+  margin-top: 80px;
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.navbar.navbar-default.navbar-fixed-top {
+  background-color: rgba(50,50,50,1);
+}
+
+.navbar-inverse .navbar-nav > li > a  {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
+  text-transform: uppercase;
+}
+
+.yield-style {
+  font-family: 'Open Sans Condensed', Arial, serif; font-weight: 400;
+  font-size: 1.5em;
+}
+
+.list {
+  margin-top: 20px;
+  margin-bottom: 60px;
+  text-align: center;
+}
+
+
+h1 {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
+  font-size: 1.5em;
+  text-align: center;
+}
+
+#logo-home > h1 {
+  font-family: 'Comfortaa', Arial, serif; font-weight: 400;
+  font-size: 2.3em;
+  font-weight: 700;
+}
+
+#order-title {
+  float: right;
+}
+
+a {
+  color: rgba(50,50,50,.5);
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Commas in function arguments should be followed by a single space